fusor-ember-cli/app/templates/rhev-options.hbs

```diff
@@ -5,10 +5,12 @@
     <br />
     <form class="form form-horizontal">
       {{text-f label="Root password for Engine & Hypervisor" type='password' value=rhev_root_password cssId='rhev_root_password' isRequired=true disabled=controllers.deployment.isStarted
+      minChars=8
       help-inline='must be 8 or more characters'
       placeholder='must be 8 or more characters'}}
 
       {{text-f label="Engine admin password" type='password' value=rhev_engine_admin_password cssId='rhev_engine_admin_password' isRequired=true disabled=controllers.deployment.isStarted
+      minChars=8
       help-inline='must be 8 or more characters'
       placeholder='must be 8 or more characters'}}
```

Comment:
@knowncitizen, in text-f.hbs, I have "must be {{minChars}} or more characters"

https://github.com/isratrade/fusor/blob/portal-login-pwd/fusor-ember-cli/app/templates/components/text-f.hbs#L14

However, for the `help-inline` and `placeholder` on a specific text field, I can't have a variable in the same place that I define `minChars`
If I do, it shows up like this in the UI and doesn't resolve to the actually number

`must be {{minChars}} or more characters`